src/Package/Repositories/Statuses.php

```diff
@@ -95,4 +95,24 @@ public function getList($user, $repo, $sha)
 		// Send the request.
 		return $this->processResponse($this->client->get($this->fetchUrl($path)));
 	}
+
+	/**
+	 * Method to get the combined Status for a specific Ref.
+	 *
+	 * @param   string  $user  The name of the owner of the GitHub repository.
+	 * @param   string  $repo  The name of the GitHub repository.
+	 * @param   string  $sha   SHA1 for which to get the combined status.
+	 *
+	 * @return  array
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getCombinedStatus($user, $repo, $sha)
+	{
+		// Build the request path.
+		$path = '/repos/' . $user . '/' . $repo . '/commits/' . $sha . '/status';
```

Comment:
I think it makes good sense to use double quotes instead of concatenating as you already did in other places

`$path = "/repos/$user/$repo/commits/$sha/status";`

I believe somewhere in some J! coding standards written documentation they say that we should not do so...